quickstart/README.md

```diff
@@ -7,17 +7,32 @@
 
 ## Lab
```

Comment:
My comments on this PR are fairly nitpicky and/or speculative. Please consider anything useful and ignore the rest.